src/Build/Resources/xlf/Strings.cs.xlf

```diff
@@ -106,6 +106,13 @@
         <note>
       {StrBegin="MSB4250: "}
       LOCALIZATION:  Do not localize the following words: ProjectGraph, ProjectReference, ToolsVersion.
+    </note>
+      </trans-unit>
+      <trans-unit id="SkippedConstraintsOnRequest">
+        <source>MSB4260: Project {0} skipped graph isolation constraints on referenced project {1}</source>
```

Comment:
I think the other messages put " around paths.

Reply:
Moved to https://github.com/microsoft/msbuild/pull/4379/files#r289096539